Directory.Build.props

```diff
@@ -12,8 +12,21 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
-    <NoWarn>$(NoWarn);NU5125</NoWarn>
+    <TargetFramework>net6.0</TargetFramework>
+    <NoWarn>$(NoWarn);NU5125;CA1416</NoWarn>
+    <!--
+      We sort of half sort of support macOS in this repo, this warning is telling us that we are on the precipice of 
```

Comment:
Opened: https://github.com/dotnet/arcade/issues/9192

We should probably reference it here.

Reply:
I think we probably want to do more than just that one project.  The x64/any disparity is all over the repo, we really need a solid plan to the dependency web here rather than "just hope we don't mess it up again". :-)